src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1068,6 +1080,68 @@
     "alter table reports add column corrective_change boolean"
     "alter table resource_events add column corrective_change boolean"))
 
+(defn resource-params-cache-parameters-to-jsonb
```

Comment:
Both of these migrations seem pretty risky. Have you done some testing on larger datasets? Seems like we could have memory issues and excessive migration times.